test/kitchen/drivers/ec2-driver.yml

```diff
@@ -102,13 +102,13 @@ platforms:
     image_id: <%= platform[1] %>
     block_device_mappings:
       <% if ENV['KITCHEN_ARCH'] == "arm64" %>
-      - device_name: /dev/sda1
+      - device_name: <%= ENV['KITCHEN_EC2_DEVICE_NAME'] || "/dev/sda1" %>
       <% else %>
-      - device_name: /dev/xvda
+      - device_name: <%= ENV['KITCHEN_EC2_DEVICE_NAME'] || "/dev/xvda" %>
       <% end %>
         ebs:
           volume_type: gp2
-          volume_size: 40
+          volume_size: 45
           delete_on_termination: true
```

Comment:
No need to fix for now: given that the device_name we need to expand entirely depends on the AMI, it would make sense to make it part of the definition of platforms done in `platforms.json`.

Reply:
do you mean to create a new machine template with a different AMI that uses the correct devices?

the problem I had, it the machines use a different device then it was before, and thus increasing the volume size had no effect on the root file system

Reply:
I meant that the [platforms.json](https://github.com/DataDog/datadog-agent/blob/main/test/kitchen/platforms.json) file could contain the information on what the root file system is (instead of having it in an env variable of the Gitlab job, which would have to be duplicated if the same image is used elsewhere), like this:

```
    "debian": {
        "ec2": {
            "arm64": {
                "debian-10": {"id": "ami-018ce69d4b9c9e3ec", "device": "/dev/xvda"},
                "debian-11": {"id": "ami-0d53a3ba77632d713", "device": "/dev/xvda"}
            }
        }
```

Since the root file system is an inherent property of the AMI, it makes sense to define them together, and that way people that want to reuse the `debian-10` image will automatically get the correct device resized, without having to set an additional env variable.

But there's no need to take care of that now, since modiying the `platforms.json` format will require some other changes to the kitchen tooling.